### PR TITLE
Rename the Abkhaz language

### PR DESCRIPTION
**Pull request by amire80**
**Phabricator:** 
https://phabricator.wikimedia.org/T298309

### Test Steps
1. Open the English Wikipedia article "Fazil Iskander".
2. Check the interlanguage links. The name of the language whose code is "ab" is supposed to be "аԥшәа". The first letter is supposed to be lowercase, and the second letter is supposed to have a tiny tail (ԥ) and not a large hook (ҧ).

### CODE REVIEW

File: Wikipedia/Code/wikipedia-languages.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   },
   {
     "languageCode" : "ab",
-    "languageName" : "Аҧсшәа",
+    "languageName" : "аԥсшәа",
```
Comment (tonisevener) [inline Wikipedia/Code/wikipedia-languages.json:9]: This languages json is occasionally generated manually from our script at WikipediaLanguagesCommandLineUtility.swift. This script internally references the site info endpoint [here](https://meta.wikimedia.org/w/api.php?action=sitematrix&smsiteprop=url%7Cdbname%7Ccode%7Csitename%7Clang&format=json&formatversion=2&origin=*), which seems to return the old spelling, so your change here will get overwritten the next time we run this. 

We have already added some workarounds for this endpoint in this script, so you could add something like this [here](https://github.com/amire80/wikipedia-ios/blob/b9082479eeb63398d295352c8ddaaba380861080/Command%20Line%20Tools/Update%20Languages/WikipediaLanguageCommandLineUtilityAPI.swift#L54) to retain this spelling. Of course ideally the site info endpoint will get updated, but this is just to give you a client-side fix option.

```
guard code != "ab" else {
    return Wikipedia(languageCode: code, languageName: "аԥсшәа", localName: localname, altISOCode: nil)
}
```

**Review by tonisevener — COMMENTED** [review]
Thanks for looking into this @amire80 ! I commented with one suggestion to make sure your changes won't get overwritten later. I also found a separate issue:

> 1. Open the English Wikipedia article "Fazil Iskander". 
> 2. Check the interlanguage links. The name of the language whose code is "ab" is supposed to be "аԥшәа". The first letter is supposed to be lowercase, and the second letter is supposed to have a tiny tail (ԥ) and not a large hook (ҧ).

Unfortunately even with your changes here I can still see this bug. It looks like the spelling for languages on this particular screen are taken from the langlinks endpoint [here](https://en.wikipedia.org/w/api.php?action=query&format=json&llinlanguagecode=en&lllimit=500&llprop=langname%7Cautonym&prop=langlinks&redirects=&titles=Fazil%20Iskander) (in the `autonym` key), which still returns the old name. I think your changes in this PR should be merged, but we'll also need that langlinks endpoint fixed in order for this bug to go away.

File: Wikipedia/Code/wikipedia-languages.json (hunk 1 above)
Comment (amire80) [reply Wikipedia/Code/wikipedia-languages.json:9]: Thanks for the comment! The name was updated in core, and that change was deployed, so the new name is seen in the code. Can you perhaps run that updating now? That letter also still appears in Wikipedia/assets/codemirror/config/codemirror-config-ab.json as part of the old linkTrail, which was also updated in core; do you update that automatically as well?

Comment (tonisevener) [reply Wikipedia/Code/wikipedia-languages.json:9]: @amire80 thanks! It's looking good, I ran the script locally and didn't see this overwritten in `wikipedia-languages.json`, so we're good there. We do generate the `codemirror-config-ab.json` file via the same script, unfortunately running it locally didn't seem to make any changes. I did some digging and it looks like it's pulling that data from [here](https://ab.wikipedia.org/w/load.php?debug=false&lang=en&modules=ext.CodeMirror.data), which is still returning the old character. If this needs to be fixed, we should also fix it at that endpoint.

**Review by tonisevener — APPROVED** [review]
> We do generate the codemirror-config-ab.json file via the same script, unfortunately running it locally didn't seem to make any changes. I did some digging and it looks like it's pulling that data from [here](https://ab.wikipedia.org/w/load.php?debug=false&lang=en&modules=ext.CodeMirror.data), which is still returning the old character. If this needs to be fixed, we should also fix it at that endpoint.

I did some digging, and I think I tracked down that last character from the CodeMirror endpoint to [this file](https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/core/+/refs/heads/master/languages/messages/MessagesAb.php#79).

I'm going to go ahead and merge this to get the other fixes in. Feel free to open up another PR here once that endpoint returns the correct characters. Thanks @amire80!

**Pull request merged**